doc/api/vm.md

```diff
@@ -94,12 +99,28 @@ changes:
     event that have been attached via `process.on("SIGINT")` will be disabled
     during script execution, but will continue to work after that.
     If execution is terminated, an [`Error`][] will be thrown.
+  * `doNotInformInspector` {boolean} If `true`, do not try to attach the
+    context to inspector. Debugging with `--inspect` will not work for code in
+    the context in that case.
 
 
 Runs the compiled code contained by the `vm.Script` object within the given
 `contextifiedSandbox` and returns the result. Running code does not have access
 to local scope.
 
+By default, this function checks if inspector was already aware of the context
+(if support for inspector is available in the Node.js build). If not, and if
+`doNotInformInspector` is not specified, this function attaches it to inspector
```

Comment:
Lots of negatives in here, which might get confusing. How about something like this:

> This function will attach the context to inspector (in Node.js builds with inspector support) using [`inspector.attachContext()`][] before running the script and detach it with [`inspector.detachContext()`][] before returning. The context will only be attached if inspector was not already aware of the context, or if `doNotInformInspector` is unspecified or `false`. Note that *it is recommended to handle context attachment and detachment manually*, since: